src/toolkits/drawing_classifier/class_registrations.cpp

```diff
@@ -5,6 +5,7 @@
  */
 #include "class_registrations.hpp"
 #include "data_preparation.hpp"
+#include "drawing_classifier.hpp"
```

Comment:
Include statements in our codebase should generally be full paths

Reply:
Done.